pkg/minikube/node/cache.go

```diff
@@ -170,7 +170,8 @@ func beginDownloadKicBaseImage(g *errgroup.Group, cc *config.ClusterConfig, down
 			}
 
 			if cc.Driver == driver.Podman {
-				return fmt.Errorf("not yet implemented, see issue #8426")
+				klog.Infof("iskipping image load from cache: not implemented for podman. see issue #8426")
```

Comment:
you can change this to warnningf

Reply:
klog.Warningf is fine, but it should go only to the log (unless --alsologtostderr was used).

Reply:
@medyagh  wrote:
> you can change this to warnningf

done.